exercises/minesweeper/example.js

```diff
@@ -0,0 +1,64 @@
+const BOMB = "*";
+
+class Minesweeper {
+  constructor() {
+    this.deltasToAdjacentSquares = [
+      [-1, -1],
+      [-1, 0],
+      [-1, 1],
+
+      [1, 1],
+      [1, 0],
+      [1, -1],
+
+      [0, 1],
+      [0, -1]
+    ]
+  }
+
+  annotate(rows) {
+    if (noDataPresent(rows)) {
+      return rows;
+    }
+
+    const inputBoard = rows.map(row => [...row]);
+    const outputBoard = [];
+    for (let x = 0; x < inputBoard.length; x++) {
```

Comment:
I was trying to think of a way to do this functionally-- it seems like an ideal case for `Array.map` since we're taking an incoming array and returning an array of the same dimensions, but with the cells transformed.